tests/JustSaying.UnitTests/JustSayingBus/WhenRegisteringPublishers.cs

```diff
@@ -4,6 +4,7 @@
 using JustSaying.Messaging;
 using JustSaying.Models;
 using JustSaying.TestingFramework;
+using Newtonsoft.Json;
```

Comment:
What makes this needed now?

Reply:
Ah, it's not. I was using this to generate the sample for the PR description and left it in. Will remove 👍

Reply:
Done